open-telemetry-addon/deploy/resources/addonmanager.role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: addon-manager
+  namespace: open-cluster-management-hub
+rules:
+- apiGroups: [""]
```

Comment:
this has to be set with a certain resourceName.

Reply:
the role is to give the permission to get the secret by the addon-manager.

Reply:
yes, but it is to give permission for monitering secret.